content/docs/manuals/cli/overview.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Overview"
```

Comment:
```suggestion
title: "Command Line Interface (CLI) Overview"
```

Reply:
I have changed it to this now - IMO this makes more sense (no unnecessary and length repetition)
plus if you do "Command Line Interface (CLI)" this gets too long and interferes with the toggle/collapse icons to the right

![image](https://github.com/user-attachments/assets/f51ffc74-c8db-4598-bf8b-7e8179596260)